install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -0,0 +1,273 @@
+==== all.yml
+
+The _all.yml_ has many options that can be modified to meet your specific needs.
+The information gathered in this file is for the provisioning portion of the instances
+required for a successful deployment of {product-title}. It
+is important to review these carefully. This document will provide a condensed 
+version of the _all.yml_ and focus on the most critical parameters that need to 
+be set for a successful deployment.
+
+
+[subs=+quotes]
+----
+*$ cat ~/inventory/group_vars/all.yml*
+---
+openshift_openstack_clusterid: "openshift"
+openshift_openstack_public_dns_domain: *"example.com"*
+openshift_openstack_dns_nameservers: *["10.19.115.228"]*
+openshift_openstack_public_hostname_suffix: "-public"
+openshift_openstack_nsupdate_zone: "{{ openshift_openstack_public_dns_domain }}"
+
+openshift_openstack_keypair_name: *"openshift"*
+openshift_openstack_external_network_name: *"public"*
+
+openshift_openstack_default_image_name: *"rhel75"*
+
+openshift_openstack_num_masters: *3*
+openshift_openstack_num_infra: *3*
+openshift_openstack_num_cns: *0*
+openshift_openstack_num_nodes: *2*
+
+openshift_openstack_master_flavor: *"m1.master"*
+openshift_openstack_default_flavor: *"m1.node"*
+
+openshift_openstack_use_lbaas_load_balancer: *true*
+
+openshift_openstack_docker_volume_size: "15"
+
+# # Roll-your-own DNS
+*openshift_openstack_external_nsupdate_keys:*
+  public:
+    *key_secret: '/alb8h0EAFWvb4i+CMA12w=='*
+    *key_name: "update-key"*
+    *key_algorithm: 'hmac-md5'*
+    *server: '<ip-of-DNS>'*
+  private:
+    *key_secret: '/alb8h0EAFWvb4i+CMA12w=='*
+    *key_name: "update-key"*
+    *key_algorithm: 'hmac-md5'*
+    *server: '<ip-of-DNS>'*
+
+ansible_user: openshift
+
+## cloud config
+openshift_openstack_disable_root: true
+openshift_openstack_user: openshift
+----
+
+NOTE: Due to using an external DNS server, the private and public sections use
+the public IP address of the DNS server as the DNS server does not reside in the
+OpenStack environment.
+
+The values above that are bold require modification based upon your OpenStack
+environment and DNS server. 
+
+In order to properly modify the DNS portion of the _all.yml_, login to the DNS
+server and perform the following commands to capture the key name, 
+key algorithm and key secret:
+
+----
+$ ssh <ip-of-DNS>
+$ sudo -i
+# cat /etc/named/<key-name.key>
+key "update-key" {
+	algorithm hmac-md5;
+	secret "/alb8h0EAFWvb4i+CMA02w==";
+};
+
+----
+
+NOTE: The key name may vary and the above is only an example. 
+
+Brief description of each variable in the table below:
+
+
+[[all_yml]]
+.Description of Variables in all.yml
+|===
+|Variable |Description
+
+|openshift_openstack_clusterid |Cluster identification name
+
+|openshift_openstack_public_dns_domain |Public DNS domain name
+|openshift_openstack_dns_nameservers | IP of DNS nameservers
+|openshift_openstack_public_hostname_suffix | Adds a suffix to the node hostname in the DNS record for both public and private
```

Comment:
openshift_openstack_public_hostname_suffix doesn’t exist any longer in “inventory/group_vars/all.yml"

Reply:
BZ opened: https://bugzilla.redhat.com/show_bug.cgi?id=1631853